.github/workflows/nightly-build-publish.yml

```diff
@@ -36,11 +53,17 @@ jobs:
       - name: Build and push happy path image
         run: /bin/bash arbitrary-users-patch/happy-path/build_happy_path_image.sh --push
 
-      - name: Build and push devfile image
+      - 
+        name: Build and push patched base images and happy path image
         run: |
-          SHORT_SHA1=$(git rev-parse --short HEAD)
-          docker build -t che-devfile-registry -f ./build/dockerfiles/Dockerfile --target registry .
-          docker tag che-devfile-registry quay.io/eclipse/che-devfile-registry:nightly
-          docker push quay.io/eclipse/che-devfile-registry:nightly
-          docker tag che-devfile-registry quay.io/eclipse/che-devfile-registry:${SHORT_SHA1}
-          docker push quay.io/eclipse/che-devfile-registry:${SHORT_SHA1}
+          /bin/bash arbitrary-users-patch/happy-path/build_happy_path_image.sh --push
```

Comment:
Both the happy path image and the base images need to be built with multi-arch support as well, otherwise the devfiles inside the registry will be no-op. I would suggest modifying the build scripts and adding `PLATFORMS` files.